### ISSUE TITLE
Create Grant Explorer main page

### ISSUE THREAD

**Issue by nfrgosselin** [gitcoinco/grants-round#348]
**User Story:**
As a round operator
I want to have a grant explorer landing page for my round
So that I have a place to send potential voters

**Acceptance Criteria**
GIVEN that I have created a round
WHEN I go to explorer.gitcoin.co/`chain_id`/`round_address`
THEN I see a blank page with the name of my program and the name of my round

**Product & Design Links:**
- plain text is fine — don't worry about formatting

**Tech Details:**
- Dependent on #354 

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

**Comment by lthrift** [comment]
I attempted to test this one but am stuck on getting a properly formatted URL. Here's as far as I got: 
https://www.loom.com/share/20408182838041d8b043de26fbaa118e

**Comment by shavinac** [comment]
The URL format is currently `https://gegitcoin.on.fleek.co/#/round/{chain_id}/{round_id}`

Example URL for Grant Explorer (staging): https://gegitcoin.on.fleek.co/#/round/5/0xa89c6f804579864e29fdbb9653f62d761d1a42fe

**Comment by lthrift** [comment]
Rejecting this one because it is pulling in the program ID number rather than the program _name_

![Image](https://user-images.githubusercontent.com/92401444/192631477-be71cd4e-8e4a-48f3-8bf2-d298fe9ed475.png)

**Comment by shavinac** [comment]
Looked into this - since **Program name** is not available directly in the **Round metadata**, retrieving the **Program name** will require an additional lookup to retrieve **Program metadata**, which is a non-trivial code change.

Before starting this work, I want to confirm that **Program name** is something we intend to display in Grant Explorer? I looked thru the [Figma](https://www.figma.com/file/lZwHrKYiWJna3AePzyTAiG/Grant-Explorer?node-id=0%3A1) and [Miro](https://miro.com/app/board/uXjVPYmqufs=/?moveToWidget=3458764532902872180&cot=14) mocks and didn't see the **Program name** anywhere.

cc @lthrift @melissa-neira @thelostone-mc

**Comment by shavinac** [comment]
Sync'd with Lindsey; if Program Name is supposed to be displayed in Grant Explorer, we will do this in a separate story

**Comment by lthrift** [comment]
accepting this story with the program ID and will create a new story if deemed necessary

**Comment by melissa-neira** [comment]
Updated design including Program Name here: https://www.figma.com/file/lZwHrKYiWJna3AePzyTAiG/Grant-Explorer?node-id=607%3A3131
Make sure to include this in the new story if needed 😎

**Comment by shavinac** [comment]
created a new issue to display program name instead of id here: https://github.com/gitcoinco/grants-round/issues/440
